<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sessionDialog</class>
 <widget class="QDialog" name="sessionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New Session Entry</string>
  </property>
  <widget class="QDateEdit" name="dateEdit">
   <property name="geometry">
    <rect>
     <x>265</x>
     <y>20</y>
     <width>115</width>
     <height>30</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::UpDownArrows</enum>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="dateLabel">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>0</y>
     <width>53</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Date</string>
   </property>
  </widget>
  <widget class="QComboBox" name="envrBox">
   <property name="geometry">
    <rect>
     <x>164</x>
     <y>20</y>
     <width>77</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="typeBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>115</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="envrLabel">
   <property name="geometry">
    <rect>
     <x>167</x>
     <y>0</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Environment</string>
   </property>
  </widget>
  <widget class="QLabel" name="typeLabel">
   <property name="geometry">
    <rect>
     <x>65</x>
     <y>0</y>
     <width>53</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Type</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="locnEdit">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>75</y>
     <width>115</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="durnEdit">
   <property name="geometry">
    <rect>
     <x>225</x>
     <y>75</y>
     <width>70</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="locnLabel">
   <property name="geometry">
    <rect>
     <x>95</x>
     <y>55</y>
     <width>53</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Location</string>
   </property>
  </widget>
  <widget class="QLabel" name="durnLabel">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>55</y>
     <width>53</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Duration</string>
   </property>
  </widget>
  <widget class="QComboBox" name="durnUnitsBox">
   <property name="geometry">
    <rect>
     <x>295</x>
     <y>75</y>
     <width>45</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="avGrBox">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>135</y>
     <width>115</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="hiGrBox">
   <property name="geometry">
    <rect>
     <x>225</x>
     <y>135</y>
     <width>115</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="avGrLabel">
   <property name="geometry">
    <rect>
     <x>75</x>
     <y>115</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Average Grade</string>
   </property>
  </widget>
  <widget class="QLabel" name="hiGrLabel">
   <property name="geometry">
    <rect>
     <x>245</x>
     <y>115</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Highest Grade</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="noteEdit">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>180</y>
     <width>321</width>
     <height>61</height>
    </rect>
   </property>
   <property name="documentTitle">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="noteLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>200</y>
     <width>53</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Note:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="submitBtn">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>260</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelBtn">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>260</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="warningLabel">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>240</y>
     <width>251</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Oops! You need to input a weight.</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="indent">
    <number>-1</number>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::NoTextInteraction</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
